lib/FindRecords/FindRecordsModal.js

```diff
@@ -27,7 +32,7 @@ import { useRecordsSelect } from './hooks';
 
 import css from './FindRecordsModal.css';
 
-export const FindRecordsModal = ({
+export const FindRecordsModal = forwardRef(({
```

Comment:
hopefully you tested it properly as it affects all our plugins

Reply:
https://github.com/user-attachments/assets/61be0e02-1798-4221-a73b-d93d86ca34c4